lib/y2x11test.pm

```diff
@@ -34,6 +34,9 @@ sub launch_yast2_module_x11 {
     type_password;
     send_key 'ret';
     assert_screen $args{target_match}, $args{match_timeout};
+    # Uses hotkey for gnome, adjust if need for other desktop
+    send_key 'alt-f10' if $args{maximize_window};
```

Comment:
Please update POD. What do we need this feature for? I assume because of the potentially long list of modules which should all fit but even in this case you can not be sure all is shown so does it really help?

Reply:
As of now it fits and that's why I've added here. We use same method for this test which is used for the registration during installation. So verifying that modules are pre-selected by single modules is not supported.